core/src/main/java/com/netflix/msl/msg/MessageInputStream.java

```diff
@@ -323,15 +323,6 @@ public MessageInputStream(final MslContext ctx, final InputStream source, final
             throw e;
         }
     }
-    
-    /* (non-Javadoc)
-     * @see java.lang.Object#finalize()
-     */
-    @Override
-    protected void finalize() throws Throwable {
-        // Do not close the source because we might want to reuse it.
-        super.finalize();
```

Comment:
I think I put this in to be explicit about not closing the source. Did this throw a FindBugs problem?